semantics.json

```diff
@@ -34,56 +34,63 @@
       "type": "group",
       "label": "Question",
       "fields": [
-        {
-          "name": "text",
-          "type": "text",
-          "widget": "html",
-          "tags": ["p", "br", "strong", "em"],
-          "label": "Text",
-          "description": "Hint for the first part of the dialogue"
-        },
-        {
-          "name": "answer",
-          "type": "text",
-          "widget": "html",
-          "tags": ["p", "br", "strong", "em"],
-          "label": "Answer",
-          "description": "Hint for the second part of the dialogue"
-        },
-        {
-          "name": "image",
-          "type": "image",
-          "label": "Image",
-          "optional": true,
-          "description": "Optional image for the card. (The card may use just an image, just a text or both)"
-        },
-        {
-          "name": "audio",
-          "type": "audio",
-          "label": "Audio files",
-          "optional": true
-        },
-        {
-          "name": "tips",
-          "type": "group",
-          "label": "Tips",
-          "fields": [
-            {
-              "name": "front",
-              "type": "text",
-              "label": "Tip for text",
-              "optional": true,
-              "description": "Tip for the first part of the dialogue"
-            },
-            {
-              "name": "back",
-              "type": "text",
-              "label": "Tip for answer",
-              "optional": true,
-              "description": "Tip for the second part of the dialogue"
-            }
-          ]
-        }
+	{
+	  "name": "text",
+	  "type": "text",
+	  "widget": "html",
+	  "tags": ["p", "br", "strong", "em"],
+	  "label": "Text",
+	  "description": "Hint for the first part of the dialogue"
+	},
+	{
+	  "name": "answer",
+	  "type": "text",
+	  "widget": "html",
+	  "tags": ["p", "br", "strong", "em"],
+	  "label": "Answer",
+	  "description": "Hint for the second part of the dialogue"
+	},
+	{
+	  "name": "image",
+	  "type": "image",
+	  "label": "Image",
+	  "optional": true,
+	  "description": "Optional image for the card. (The card may use just an image, just a text or both)"
+	},
+	{
+	  "name": "backImage",
```

Comment:
backImage needs translations to all languages (you don't actually have to translate them, just make sure the languages structures are correct)